contracts/contracts/initialVoiceCreditProxy/InitialVoiceCreditProxy.sol

```diff
@@ -1,6 +1,14 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.10;
 
+/// @title InitialVoiceCreditProxy
+/// @notice This contract is the bast contract for
```

Comment:
what does it mean by **the bast contract**?

Reply:
was meant to be "base" - good catch